packages/mdctl-node-utils/crypto.js

```diff
@@ -22,6 +22,20 @@ function md5FileHash(filename) {
   return hash.digest('hex')
 }
 
+function hashCode(str, seed = 0) {
```

Comment:
When I said you could use crypto I mean you could use this 
```
const hash = createHash('md5')
hash.update(str)
return hash.digest('hex')
```

Reply:
OK, I'll change it to this